packages/vue/src/tracing.ts

```diff
@@ -40,7 +41,7 @@ function finishRootSpan(vm: VueSentry, timestamp: number, timeout: number): void
   }
 
   vm.$_sentryRootSpanTimer = setTimeout(() => {
-    if (vm.$root?.$_sentryRootSpan) {
+    if (vm.$root.$_sentryRootSpan) {
```

Comment:
Also here just to double check cc @Lms24 , type-wise `$root` cannot be undefined, not sure if the optional chaining guard was there for a different reason?

Reply:
Hmm not sure to be honest. With the type, you mean this here, correct?
https://github.com/getsentry/sentry-javascript/blob/2aa4e94b036675245290596884959e06dcced044/packages/vue/src/types.ts#L14-L27

Since it seems like we vendored this type from Vue, I wouldn't be too sure that `$root` always existed (older Vue versions...). According to this PR, we introduced it with Vue 3 support: https://github.com/getsentry/sentry-javascript/pull/3804. I'd just check for `vm.$root` here anyway to make sure. Shouldn't hurt us too much in terms of bundle size. WDYT?

Reply:
jup, I will revert this then, to be safe!

Reply:
I checked, and as far as I can tell $root _should_ be available in both vue2 and vue3. But, hard to be sure... so I reverted the check, to be on the safe side.